internal/cli/opsmanager/versionmanifest/update_test.go

```diff
@@ -56,6 +59,41 @@ func TestVersionManifestUpdate_Run(t *testing.T) {
 	}
 }
 
+func TestVersionManifestUpdate_Run_BadFormat(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockVersionManifestGetterUpdater(ctrl)
+	defer ctrl.Finish()
+
+	updateOpts := &UpdateOpts{
+		versionManifest: "bad version",
+		store:           mockStore,
+	}
+
+	if err := updateOpts.Run(); err == nil {
+		t.Fatalf("Run() expected error to be returned")
+	}
+}
+
+func TestVersionManifestUpdate_Run_InvalidOMVersion(t *testing.T) {
```

Comment:
[not blocking] I prefer to use a table approach, check https://github.com/mongodb/mongocli/blob/41c3a3c95380aa7d8f896b0e82868ccce2dcff30/internal/cli/alerts/acknowledge_test.go#L52-L86